ui/components/Filters.js

```diff
@@ -1450,61 +1449,50 @@ const ImportModal = React.memo((props) => {
   // const classes = useStyles();
 
   return (
-    <UsesSistent>
-      <SistentModal
-        open={true}
-        closeModal={handleClose}
-        headerIcon={
-          <Filter
-            fill="#fff"
-            style={{ height: '24px', width: '24px', fonSize: '1.45rem' }}
-            className={undefined}
-          />
-        }
-        title="Import Design"
-        maxWidth="sm"
-      >
-        <RJSFModalWrapper
-          schema={importFormSchema.rjsfSchema}
-          uiSchema={importFormSchema.uiSchema}
-          handleSubmit={handleImportFilter}
-          submitBtnText="Import"
-          handleClose={handleClose}
+    <Modal
+      open={true}
+      handleClose={handleClose}
+      headerIcon={
+        <Filter
+          fill="#fff"
+          style={{ height: '24px', width: '24px', fonSize: '1.45rem' }}
+          className={undefined}
```

Comment:
The purpose of including "undefined" is to ensure that another name isn't accidentally used? Or... what is the purpose of this?

Reply:
No it doesn't have any other purpose and also its not needed because we are not required to pass any classname to this . I have removed and passed theme and proper props